buildSrc/updateLibs.js

```diff
@@ -63,6 +70,20 @@ async function copyToLibs(files) {
 	}
 }
 
+function updateVendoredVersionsFile() {
+	const names = clientDependencies
+		.map((d) => (typeof d === "string" ? d : d.src))
+		.map(path.normalize)
+		.map((srcPath) => srcPath.split(path.sep)[2])
+	const versions = JSON.parse(fs.readFileSync(path.join(__dirname, "..", "package.json"), "utf-8")).dependencies
```

Comment:
we do read versions from package-json multiple times but I am wondering if we should even take the resolved versions with `npm list` or such (would also make it more reliable if we remove `npm i` here)